petab/petablint.py

```diff
@@ -162,6 +162,8 @@ def main():
             sys.exit(1)
 
     ret = petab.lint.lint_problem(problem)
+    if not ret:
+        print('PEtab format check completed successfully')
```

Comment:
When running `petablint` without `-v`, there is no output on purpose. I'd like to leave that as is.

If you are running `petablint -v`, you currently get an `OK` in case of no problems. If you think that is too uninformative, it's okay to put the message there (end of `petab.lint.lint_problem`).

Reply:
looks good. That was, what I had in mind :)